auth/util/src/main/java/org/wildfly/security/auth/util/RegexNameRewriter.java

```diff
@@ -34,12 +35,12 @@
     /**
      * Construct a new instance.
      *
-     * @param pattern the substitution pattern
+     * @param pattern the substitution pattern (must not be {@code null})
      * @param replacement the replacement string
      * @param replaceAll {@code true} to replace all occurrences of the pattern; {@code false} to replace only the first occurrence
      */
     public RegexNameRewriter(final Pattern pattern, final String replacement, final boolean replaceAll) {
-        this.pattern = pattern;
+        this.pattern = Assert.checkNotNullParam("pattern", pattern);;
```

Comment:
@lvydra Thank you for PR! just a minor, there are 2 `;` here, which makes trouble for checkstyle